docs/api/basic/useSnapshot.mdx

```diff
@@ -26,7 +31,65 @@ function Counter() {
 }
 ```
 
-#### Read only what you need
+### Parent/Child Components
+
+If you have a parent component use `useSnapshot`, it can pass snapshots to child components and the parent & children will re-render when the snapshot changes.
+
+For example:
+
+```jsx
+const state = proxy({
+  books: [
+    { id: 1, title: 'b1' },
+    { id: 2, title: 'b2' },
+  ],
+})
+
+function AuthorView() {
+  const snap = useSnapshot(state)
+  return (
+    <div>
+      {snap.books.map((book) => (
+        <Book key={book.id} book={book} />
+      ))}
+    </div>
+  )
+}
+
+function BookView({ book }) {
+  // book is a snapshot
+  return <div>{book.title}</div>
+}
+```
+
+If book 2's title is changed, a new `snap` is created and the `AuthorView` and `BookView` components will re-render.
+
+Note if `BookView` is `React.memo`d, the 1st `BookView` will not re-render, b/c the 1st `Book` snapshot will be the same instance, as only the 2nd `Book` was mutated (the root `Author` snapshot will also be updated since the list of `books` has changed).
+
+### Child Components Making Mutations
+
+The above approach works if `BookView` is read-only; if your child component needs to make mutations, then you'll need to pass the store:
```

Comment:
btw, we could pass both snap and state.

Reply:
Sounds good, I pushed an update with a "passing both" example.